doc/api-extensions.md

```diff
@@ -2230,3 +2230,6 @@ When the offline cluster member is evacuated, only remote-backed instances will
 
 ## `instances_state_total`
 This extension adds a new `total` field to `InstanceStateDisk` and `InstanceStateMemory`, both part of the instance's state API.
+
+## `instances_rebuild`
```

Comment:
commit should start "api: " like the other ones:

https://github.com/lxc/lxd/commits/master/doc/api-extensions.md